### PR TITLE
Add coverage report

### PR DESCRIPTION
**Pull request by camjc**
#### What does this PR accomplish?
Output coverage report in lcov and html when running karma.

#### How should this be manually tested?
- [x] Check that /coverage exists and shows coverage for each existing component.

#### Any background context you want to provide?
- Coverage report is of transpiled output instead of the ES2015 output.
  Change this once better tooling is available.
- Still TODO: completely cover slicey components.

#### What are the relevant trello cards?
https://trello.com/c/1padAmYA/4506-geopicker-react-components-es6-repo-5d

#### Screenshots (if appropriate)
screen shot 2015-10-26 at 1 05 57 pm

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -14,9 +14,15 @@ module.exports = function configureKarma(config) {
...
-    reporters: ['mocha'],
+    reporters: ['coverage', 'mocha'],
     preprocessors: {
-      'test/loadtests.js': ['webpack', 'sourcemap'],
+      'test/loadtests.js': ['webpack'],
+    },
+    coverageReporter: {
+      dir: 'coverage/',
+      reporters: [
+        { type: 'lcov', subdir: '.'},
```
Comment (runk) [inline karma.conf.js]: {type

**Comment by runk** [comment]
LGTM

**Comment by camjc** [comment]
Cheers @runk, added a linting rule to auto-fix mistakes like that.

**Comment by runk** [comment]
Merging?

**Comment by camjc** [comment]
@runk Yes please.

**Comment by omgaz** [comment]
Looks good

**Pull request merged**